### PR TITLE
Feature :Alert component

### PR DESCRIPTION
**Pull request by kiranpratyush**
updated index.html according to the documentation site(though some changes are pending for header ignore it for now)
updated the style.css
Below is the live working link please review it and give help full feedback

Components Defined - 
- Alert

## Live Link - 
Documentation site : [Testing](https://testing2311.netlify.app/alert/)

### CODE REVIEW

File: Alert/index.html [hunk 1]
```diff
@@ -1,18 +1,99 @@
...
+          <div class="alert alert-secondary">I am a Secondary Alert</div>
+        </div>
+        <div
+          class="container iframe"
+          style="
+            height: 300px;
+            background-color: transparent;
+            padding-left: 0px;
+            padding-right: 0px;
+          "
```
Comment (gopalbbharadva) [inline Alert/index.html:83]: Suggestion: Inline styles are bad practice so your can group it in a class.

**Review by gopalbbharadva — APPROVED** [review]

File: Alert/index.html (hunk 1 above)
Comment (kiranpratyush) [reply Alert/index.html:83]: Iframe is behaving weirdly so everywhere styling is changed I am planning to remove it 😂 But after first Submission
Thank you for your valuable feedback

**Review by srishti-maurya — APPROVED** [review]
looks good

**Comment by kiranpratyush** [comment]
> looks good

Thank you

**Pull request merged**